src/dkpy/structured_singular_value.py

```diff
@@ -7,14 +7,21 @@
 
 import abc
 import warnings
-from typing import Any, Dict, Optional, Tuple
+from typing import Any, Dict, Optional, Tuple, Union, Sequence
 
 import cvxpy
 import joblib
 import numpy as np
 import scipy.linalg
 
 from . import utilities
+from .uncertainty_structure import (
```

Comment:
`from . import uncertainty_structure` (just to match the way I did it everywhere else)